src/main/java/Duke.java

```diff
@@ -34,35 +34,30 @@
     private Image user = new Image(this.getClass().getResourceAsStream("/images/DaUser.png"));
     private Image duke = new Image(this.getClass().getResourceAsStream("/images/DaDuke.png"));
 
-//    public static void main(String[] args) throws Exception {
-//        new Duke("data/duke.txt");
-//    }
+    public static void main(String[] args) throws Exception {
+
```

Comment:
I think main( ) should not be empty, and also shouldn't be throwing an Exception if it's empty.